web_console/dynamic-plugin/overview-dynamic-plugin.adoc

```diff
@@ -39,11 +39,15 @@ conster Header: React.FC = () => {
 ----
 
 * Avoid selectors that could affect markup outside of your plugins components, such as element selectors. These are not APIs and are subject to change. Using them might break your plugin. Avoid selectors like element selectors that could affect markup outside of your plugins components.
+* Provide valid JavaScript Multipurpose Inernet Mail Extension (MIME) type using the `Content-Type` response header for all assets served by your plugin web server. Each plugin deployment should include a web server that hosts the generated assets of the given plugin.
 
 [discrete]
 == PatternFly guidelines
 When creating your plugin, follow these guidelines for using PatternFly:
 
 * Use link:https://www.patternfly.org/components/all-components/[PatternFly] components and PatternFly CSS variables. Core PatternFly components are available through the SDK. Using PatternFly components and variables help your plugin look consistent in future console versions.
+** Use Patternfly 4.x if you are using {product-title} versions 4.14 and before.
```

Comment:
Per IBM Style Guide, it should read "using {product-title} 4.14 and earlier."

See: https://www.ibm.com/docs/en/ibm-style?topic=references-product-versions#references-to-earlier-or-later-versions